ros_ws/src/crazyswarm/scripts/test_collisionAvoidance.py

```diff
@@ -42,9 +44,123 @@ def test_velocityMode_sidestepWorstCase(args=None):
+    cf1.goTo([-1.0, 0.0, 1.0], 0, 5.0)
+    # flag if a collision happened during sleep
+    collisionHappend = False
+    while timeHelper.time() < 10.0:
+        positions = np.stack([cf0.position(), cf1.position()])
+        collisions = check_ellipsoid_collisions(positions, RADII)
+        if np.any(collisions):
+            collisionHappend = True
+
+        timeHelper.sleep(timeHelper.dt)
+        if cf0.position()[0] > 1.0 and cf1.position()[0] < -1.0:
+            return
+    assert collisionHappend
+
+
+def test_goToWithCA_CheckCollision():
+    args = "--sim --vis null"
+    allcfs, timeHelper = setUp(args)
+    allcfs.takeoff(targetHeight=Z, duration=1.0+Z)
+    timeHelper.sleep(1.5+Z)
+    cf0, cf1 = allcfs.crazyflies
+    cf0.enableCollisionAvoidance([cf1], RADII)
+    cf1.enableCollisionAvoidance([cf0], RADII)
+
+    cf0.goTo([1.0, 0.0, 1.0], 0, 5.0)
+    cf1.goTo([-1.0, 0.0, 1.0], 0, 5.0)
+    while timeHelper.time() < 10.0:
+        positions = np.stack([cf0.position(), cf1.position()])
+        collisions = check_ellipsoid_collisions(positions, RADII)
+        assert not np.any(collisions)
+
+        timeHelper.sleep(timeHelper.dt)
+        if cf0.position()[0] > 1.0 and cf1.position()[0] < -1.0:
+            return
+
+
+def test_goToWithCA_CheckDestination():
+    args = "--sim --vis null"
+    allcfs, timeHelper = setUp(args)
+    allcfs.takeoff(targetHeight=Z, duration=1.0+Z)
+    timeHelper.sleep(1.5+Z)
+    cf0, cf1 = allcfs.crazyflies
+    cf0.enableCollisionAvoidance([cf1], RADII)
+    cf1.enableCollisionAvoidance([cf0], RADII)
+
+    goal0 = [1.0, 0.0, 1.0]
+    goal1 = [-1.0, 0.0, 1.0]
+    cf0.goTo(goal0, 0, 5.0)
+    cf1.goTo(goal1, 0, 5.0)
+    timeHelper.sleep(5)
+    assert np.all(np.isclose(cf0.position(), goal0))
+    assert np.all(np.isclose(cf1.position(), goal1))
+
+
+def test_goToWithCA_changeEllipsoid():
+    args = "--sim --vis null"
+    allcfs, timeHelper = setUp(args)
+    allcfs.takeoff(targetHeight=Z, duration=1.0+Z)
+    timeHelper.sleep(1.5+Z)
+    cf0, cf1 = allcfs.crazyflies
+
+    newRADII = np.array([0.1, 0.1, 0.3])
+    cf0.enableCollisionAvoidance([cf1], newRADII)
+    cf1.enableCollisionAvoidance([cf0], newRADII)
+
+    goal0 = [1.0, 0.0, 1.0]
+    goal1 = [-1.0, 0.0, 1.0]
+    cf0.goTo(goal0, 0, 5.0)
+    cf1.goTo(goal1, 0, 5.0)
+
+    # flag if a collision happened during sleep
+    collisionHappend = False
+    while timeHelper.time() < 10.0:
+        positions = np.stack([cf0.position(), cf1.position()])
+        collisionsOld = check_ellipsoid_collisions(positions, RADII)
+        collisionsNew = check_ellipsoid_collisions(positions, newRADII)
+        if np.any(collisionsOld):
+            collisionHappend = True
+        assert not np.any(collisionsNew)
+
+        timeHelper.sleep(timeHelper.dt)
+        if cf0.position()[0] > 1.0 and cf1.position()[0] < -1.0:
+            return
+    assert collisionHappend
+
+
+def test_cmdPosition():
+    args = "--sim --vis null --dt 0.01"
```

Comment:
we should add the `maxvel` arg whenever using `cmdPosition` like this - otherwise the CFs will jump straight to the goal position